### PR TITLE
[Backend] Event service

### PR DESCRIPTION
**Pull request by enozcan**
The PR contains the service for economic events. Events are fetched from `tradingeconomics` 3rd party service and updated at 8 A.M (UTC+3) everyday. Since no further operations concerning events (e.g. comment, follow, like etc.) are required, the system only stores the json data itself coming from the service. Thus, there is no explicit model for economic events. Also, history is not being saved. Only that day's events are served.

The original response can be seen [here.](https://api.tradingeconomics.com/calendar?c=guest:guest&format=json)

Endpoint: `[GET] /events`

Response: (as String) 
```json
[
    {
        "CalendarId": "196769",
        "Date": "2019-12-15T11:10:00",
        "Country": "Israel",
        "Category": "Current Account",
        "Event": "Current Account",
        "Reference": "Q3",
        "Source": "Central Bureau of Statistics, Israel",
        "Actual": "$3.9B",
        "Previous": "$4.2B",
        "Forecast": "",
        "TEForecast": "$3.2B",
        "URL": "/israel/current-account",
        "DateSpan": "0",
        "Importance": 1,
        "LastUpdate": "2019-12-15T11:10:00",
        "Revised": "",
        "Currency": "",
        "Unit": "",
        "OCountry": "Israel",
        "OCategory": "Current Account",
        "Ticker": "ISCAL",
        "Symbol": "ISCAL"
    },
    {
        ...
    }
   
]
```

### CODE REVIEW

**Comment by enozcan** [comment]
@burakikanyildiz It's only thrown when there is no events data saved to database (either up-to-date or belongs to previous days). During application start-up, these events are supposed to be initialized if there is no such events data and this initialization is performed using the data fetched from 3rd party. The problem occurs only if the 3rd party service fails to send response to our server on start-up. In such a scenario, we can force our server to initialize them using `/events/force` endpoint which can be reached by admin user only. 

So, in short, there is no restriction to use 3rd party API. The `INTERNAL_SERVER_ERROR` response is to notify that something went wrong during initialization and force init is required.

**Comment by barisegesevgili** [comment]
All looks good to me as well. Besides, tradingeconomics wont annoy us as an API service IMO. Thanks!
